AABB_tree/include/CGAL/AABB_traits.h

```diff
@@ -180,12 +180,15 @@ class AABB_traits
 {
   typedef typename CGAL::Object Object;
 public:
+
   typedef GeomTraits Geom_traits;
+  typedef AABBPrimitive Primitive;
+  typedef BboxMap Bbox_map;
+  typedef CGAL::Parallel_tag Concurrency_tag;
```

Comment:
This header should not be modified, otherwise it means the concept is not longer correct

Reply:
Related to https://github.com/CGAL/cgal/pull/5893#discussion_r688562692, this was necessary to work around [a limitation of template type deduction](https://deque.blog/2017/10/12/why-template-parameters-of-dependent-type-names-cannot-be-deduced-and-what-to-do-about-it/).